src/components/app/data/hooks/useEnterpriseFeatures.js

```diff
@@ -5,13 +5,14 @@ export default function useEnterpriseFeatures(queryOptions = {}) {
   return useEnterpriseLearner({
     ...queryOptionsRest,
     select: (data) => {
+      const transformedData = data.transformed.enterpriseFeatures;
       if (select) {
         return select({
           original: data,
```

Comment:
nit: similar comment here around where we should pass `data` or `data.original` (curious what you think)